src/schema/search.ts

```diff
@@ -310,7 +310,7 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
       const limit = Math.min(args.first || 10);
       const offset = getOffsetWithDefault(args.after, -1) + 1;
       const meilieSearchRes = await searchMeili(
-        `q=${args.query}&attributesToRetrieve=post_id&attributesToSearchOn=title&limit=${limit}&offset=${offset}`,
+        `q=${args.query}&attributesToRetrieve=post_id${args.version === 2 && '&attributesToSearchOn=title'}&limit=${limit}&offset=${offset}`,
```

Comment:
I think you need `args.version === 2 ? '&attributesToSearchOn=title' : ''`, because like this you will write `'false'`

Maybe good time to move to `URLSearchParams`, non blocking.